makefile

```diff
@@ -240,7 +241,9 @@ install_DIRS += $(_DESTDIR)$(sysconfdir)/cron.daily
 #
 default_INSTCMD = cp
 $(_DESTDIR)$(sysconfdir)/cron.daily_TARGETS        = pkg/cron.daily:serviced
+$(_DESTDIR)$(sysconfdir)/cron.hourly_TARGETS       = pkg/cron.hourly:serviced
 $(_DESTDIR)$(prefix)/etc_TARGETS                   = pkg/serviced.logrotate:logrotate.conf
+$(_DESTDIR)$(prefix)/etc_TARGETS                  += pkg/logrotate-docker.conf:logrotate-docker.conf
```

Comment:
Yeah, that's fine. I'm leery of doing things to the host system that might conflict with normal administrator stuff, and logrotate.d exists for a reason. We can do an out-of-band logrotate though, I guess.